### ISSUE TITLE
new build causing error on meteor run

### ISSUE THREAD

**Issue by gjolund** [huttonr/bootstrap3#10]
I just uninstalled and reinstalled your package and am not getting the following error on meteor run

``` shell
W20160119-23:02:57.173(-8)? (STDERR)          
W20160119-23:02:57.175(-8)? (STDERR) /Users/user/code/hsd_admin/.meteor/local/build/programs/server/packages/huttonr_bootstrap3.js:72
W20160119-23:02:57.175(-8)? (STDERR) `                                                                             
W20160119-23:02:57.175(-8)? (STDERR) ^
W20160119-23:02:57.175(-8)? (STDERR) SyntaxError: Unexpected token ILLEGAL
W20160119-23:02:57.175(-8)? (STDERR)     at /Users/user/code/hsd_admin/.meteor/local/build/programs/server/boot.js:241:30
W20160119-23:02:57.176(-8)? (STDERR)     at Array.forEach (native)
W20160119-23:02:57.176(-8)? (STDERR)     at Function._.each._.forEach (/Users/user/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160119-23:02:57.176(-8)? (STDERR)     at /Users/user/code/hsd_admin/.meteor/local/build/programs/server/boot.js:137:5
```

**Comment by huttonr** [comment]
@austinrivas 
This is a very odd error considering the fact that `.meteor/local/build/programs/server/packages/huttonr_bootstrap3.js` should be 19 lines long and yet the messages says the error occurs on line 72.

I'll need some more data.  What version of meteor is the project running on?  What other packages or dependencies are being used?  Are you sure you are using `huttonr:bootstrap3@3.3.6_5` (that version specifically)?  Based on your paths you are running OS X right?  Does `huttonr:bootstrap3@3.3.6_3` work?  Have you tried a `meteor reset` (which, **warning**, will delete any local database entries stored in mongo)?

Please let me know, I'm eager to assist.

**Comment by solace** [comment]
Same issue, in packages added `huttonr:bootstrap3@3.3.6_5` with and without the version and see this error as well. `6_3` also gives the same error.

Have removed all packages aside from that which came with core init, and `meteor reset` and the same thing happens.

**Comment by solace** [comment]
For reference, that file is a few hundred lines long.

It appears to be a combination of `packages/huttonr_bootstrap3/check-settings.js` and `packages/huttonr_bootstrap3/plugin/bootstrap3.js` and 72 in the error is the backtick here: https://github.com/huttonr/bootstrap3/blob/master/plugin/bootstrap3.js#L26

**Comment by huttonr** [comment]
Looks like the ES 6 stuff wasn't being compiled.  I reimplemented ecmascript and this is now fixed with
[huttonr:bootstrap3@3.3.6_6](0329bff49c757c3ce0f62dd838185b3127c52e20).

Please let me know that it's solved now.

**Comment by huttonr** [comment]
@austinrivas 
Just wanted to verify that this issue is now solved with `huttonr:bootstrap3@3.3.6_6`

**Comment by gjolund** [comment]
great, thanks for the update @huttonr
